app/views/cards/new.html.erb

```diff
@@ -1,2 +1,5 @@
 <h1>Cards#new</h1>
 <p>Find me in app/views/cards/new.html.erb</p>
+<div class="btn btn-success">
+  <%= link_to 'Write your message!', edit_card_path(@card), :class => "btn" %>
```

Comment:
You don't need a div to apply bootstrap style to a link_to. You can add the btn-success directly in :class =>"btn"
instead of :class => writing class: 'btn btn-success' should work and more readable.

Reply:
ok will make it better :) it thx actually, I was thinking to add a simple for instead as it has a button incuded